sci-libs/opencascade/opencascade-7.5.1-r1.ebuild

```diff
@@ -80,6 +80,12 @@ src_prepare() {
 
 	sed -e 's/\/lib\$/\/'$(get_libdir)'\$/' \
 		-i adm/templates/OpenCASCADEConfig.cmake.in || die
+
+	# There is an OCCT_UPDATE_TARGET_FILE cmake macro that fails due to some
+	# assumptions it makes about installation paths. Rather than fixing it, just
+	# get rid of the mechanism altogether - its purpose is to allow a
+	# side-by-side installation of release and debug libraries.
+	sed -e 's|\${OCCT_INSTALL_BIN_LETTER}||' -i "${BUILD_DIR}/"
```

Comment:
Hmm...
```
sed: couldn't edit /var/tmp/portage/sci-libs/opencascade-7.5.1-r1/work/opencascade-7.5.1_build/: not a regular file
```

Reply:
yikes, I get the same. wonder why that's happening now but not earlier when I submitted the PR....

either way, I'll fix it.